packages/DropdownMenu/src/ContentContainer.styles.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const ContentContainerStyles = styled.div`
+  background: ${tokens.color.white};
+  border: 1px solid ${tokens.border.color};
+  border-radius: ${tokens.popover.borderRadius};
+  box-shadow: ${tokens.popover.shadow};
+  margin-left: ${stylers.spacer(2)};
+  max-width: 320px;
```

Comment:
Changed this `max-width` to match `max-width` that is the default in Popover content component

Reply:
Interesting... maybe we should make a token?

Reply:
Sure, that could be a good idea